GeoLib/AABB.h

```diff
@@ -85,6 +85,20 @@ class AABB
 		}
 	}
 
+	/**
+	 * Construction of object using input pointer array.
+	 * @param pnt_array  an array of point pointers
+	 * @param size       size of the array
+	 */
+	AABB(PNT_TYPE * const* pnt_array, unsigned size) :
```

Comment:
Instead using this constructor I think it is possible to use the constructor in line 72.